python/py-jupytext/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        mwouts jupytext 1.4.0
+
+name                py-jupytext
+revision            0
+categories-append   devel
+maintainers         {gmail.com:jjstickel @jjstickel} openmaintainer
+platforms           darwin
+license             MIT
+supported_archs     noarch
+
+description         Jupyter Notebooks as Markdown Documents, Julia, Python or R Scripts
+
+long_description    Jupytext can save Jupyter notebooks as Markdown and R \
+                    Markdown documents and Scripts in many languages. It can \
+                    also convert these documents **into** Jupyter Notebooks, \
+                    allowing you to synchronize content in both directions.
+
+checksums           rmd160  9502fef40d095a2c5f04a882ce2aea1b179cd78a \
+                    sha256  8184375aa68d8b3be1e1664aa9b49f418376471ffadadc3e617b96f2575a9511 \
+                    size    4322795
+
+python.versions     36 37 38
+python.default_version      37
```

Comment:
why not let the python PG take care of the default setting? I would prefer to have this line removed and use Python 3.8 as is set as the default in the PG no.

Reply:
I've made this change. I wasn't aware that the portgroup sets a default subport. Thanks for pointing this out.